tests/test_class_schema.py

```diff
@@ -0,0 +1,34 @@
+import dataclasses
+import unittest
+
+from marshmallow_dataclass import class_schema
+
+
+class TestClassSchema(unittest.TestCase):
+
+    def test_simple_unique_schemas(self):
+        @dataclasses.dataclass
+        class Simple:
+            one: str = dataclasses.field()
+            two: str = dataclasses.field()
+
+        @dataclasses.dataclass
+        class ComplexNested:
+            three: int = dataclasses.field()
+            four: Simple = dataclasses.field()
+
+        self.assertEqual(id(class_schema(ComplexNested)), id(class_schema(ComplexNested)))
+        self.assertTrue(class_schema(ComplexNested) is class_schema(ComplexNested))
```

Comment:
Why not [`assertIs`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIs) ?